src/UniTask/Assets/Plugins/UniTask/Runtime/UniTask.Delay.cs

```diff
@@ -75,6 +75,12 @@ public static UniTask NextFrame(PlayerLoopTiming timing, CancellationToken cance
             return new UniTask(NextFramePromise.Create(timing, cancellationToken, out var token), token);
         }
 
+#if UNITY_2023_1_OR_NEWER
+        public static async UniTask WaitForEndOfFrame(CancellationToken cancellationToken = default)
```

Comment:
I have done a easy implementation that uses `async`.
Should I consider adding Awaitable.ToUniTask()?

Reply:
OK to use async.